mobile/App.js

```diff
@@ -8,12 +8,12 @@
 
 import React from 'react';
 import {SafeAreaProvider} from 'react-native-safe-area-context';
-import ScanView from './src/views/ScanView';
+import HomeView from './src/views/HomeView';
 
 const App = () => {
   return (
     <SafeAreaProvider>
-      <ScanView></ScanView>
+      <HomeView></HomeView>
```

Comment:
I'm gonna assume you'll bring back ScanView in the ticket where you do the bottom navbar

Reply:
It is what I am going to do during the navigation ticket